modules/core/src/main/java/org/apache/ignite/internal/util/IgniteUtils.java

```diff
@@ -4217,6 +4217,32 @@ public static void close(@Nullable AutoCloseable rsrc, @Nullable IgniteLogger lo
         }
     }
 
+    /**
+     * Closes given socket logging possible checked exception.
+     *
+     * @param sock Socket to close. If it's {@code null} - it's no-op.
+     * @param log Logger to log possible checked exception with (optional).
+     */
+    public static void close(@Nullable Socket sock, @Nullable IgniteLogger log) {
+        if (sock != null) {
+            try {
+                //avoid tls 1.3 incompatibility https://bugs.openjdk.java.net/browse/JDK-8208526
+                sock.shutdownOutput();
+                sock.shutdownInput();
+            }
+            catch (Exception e) {
+                warn(log, "Failed to close socket: " + e.getMessage(), e);
```

Comment:
I suggest to use different log message here, like `Failed to shutdown socket: `, it will make logs analysis a bit easier.

Reply:
done